spec/features/unlinking_a_defendant_from_maat_spec.rb

```diff
@@ -4,8 +4,8 @@
 
 RSpec.feature 'Unlinking a defendant from MAAT', :stub_unlink_v2, type: :feature do
   let(:case_urn) { 'TEST12345' }
-  let(:api_url_v2) { CdApi::BaseModel.site }
-  let(:api_request_path) { "#{api_url_v2}laa_references/#{defendant_id}/" }
+  let(:api_url_v2) { "http://localhost:9292/api/internal/v2/" }
```

Comment:
this is your local setting,  we should use an ENV var, I am using this on `.env.development.local`:

`COURT_DATA_ADAPTOR_API_V2_URL=http://localhost:3000/api/internal/v2 `

(also, in the branch for the Hearing I am working on, I created the `Cda::BaseModel` , I'll open a draft pr, still having problem with testing)